src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -87,7 +87,7 @@ static size_t _symbols_removed = 0;
 static size_t _symbols_counted = 0;
 static size_t _current_size = 0;
 
-static volatile size_t _items_count = 0;
+static volatile int    _items_count = 0;
 static volatile bool   _has_items_to_clean = false;
```

Comment:
Nit: the extra spaces are not needed now we use a shorter type name

Reply:
Since I'm reverting the change, it can have extra spaces.